front_end/src/components/charts/multiple_choice_chart.tsx

```diff
@@ -185,6 +186,48 @@ const MultipleChoiceChart: FC<Props> = ({
       }}
     />
   );
+
+  // Define a custom "X" symbol function
+  const GetSymbol: React.FC<PointProps> = (props: PointProps) => {
```

Comment:
That's a bad practice to define react components inside other component body. Let's move it outside (can be the same file right after the `buildChartData`).

Additionally, it would be better to rename this component to something like `PredictionSymbol` or something similar.